### ISSUE TITLE
Headings: H5 zonder marges in Info en Rich Content

### ISSUE THREAD

**Issue by hansgrimm** [dso-toolkit/dso-toolkit#2110]
Wens is om een Heading 5 (dat is `h5` en `.h5`) boven een blok tekst te plaatsen, zonder dat daar een marge tussen zit.

H5 moet op de volgende locaties geen marges krijgen waardoor ze dichter op de tekst staan:
* H5 in Rich Content in Action List Item
* H5 in Info in Form
![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/2361f3be-101d-410a-bebc-8dadcb5bea9b)

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/606c8127-591c-451e-9eca-d633b32270f9)

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/05afc062-aefe-4422-baa5-2e6641adfb7b)

**Comment by tfrijsewijk** [comment]
@MYpenburg kan jij mijn wijzigingen in het issue bevestigen?

@LukeVerlaan Ik denk dat je vast een voorzet kan doen hoor, dit is allemaal stukken simpeler dan wat je eerst moest maken.

**Comment by tfrijsewijk** [comment]
Via de Slack heeft Martin bovenstaand lijstje bevestigd.
